robyn/processpool.py

```diff
@@ -1,6 +1,9 @@
 import asyncio
+from multiprocessing import Process
```

Comment:
I'd recommend you doing the following

replace it with `from multiprocess import Process`.

We have the following library as a dependency that allows better serialization - https://pypi.org/project/multiprocess/

Reply:
We should be importing this `__init__.py` too. I am unsure why we are using the default `multiprocessing` there.

Reply:
Good catch, I didn't see this one!

Reply:
I'll change it in __init__ too then